### PR TITLE
Bind input plugin

### PR DESCRIPTION
**Pull request by danielllek**
This PR is a follow up on https://github.com/influxdata/telegraf/pull/2383

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

**Comment by danielllek** [comment]
@danielnelson I think it's complete now, tests fixed and passed :)

File: plugins/inputs/bind/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+1553863200000000000 315.64999999990687        2128.483333332464
+1553863500000000000 308.9166666667443         2132.350000000559
+1553863800000000000 302.64999999990687        2131.1833333335817
+1553864100000000000 310.85000000009313        2132.449999999255
+1553864400000000000 314.3666666666977         2136.216666666791
+1553864700000000000 303.2333333331626         2133.8166666673496
+1553865000000000000 304.93333333334886        2127.333333333023
+1553865300000000000 317.93333333334886        2130.3166666664183
+1553865600000000000 280.6666666667443         1807.9071428570896
+```
```
Comment (danielnelson) [inline plugins/inputs/bind/README.md]: Could you add a Example Output section at the bottom like in the [EXAMPLE_README](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/EXAMPLE_README.md)?

File: plugins/inputs/bind/xml_stats_v3.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+			return err
+		}
+
+		defer resp.Body.Close()
+
+		if resp.StatusCode != http.StatusOK {
+			return fmt.Errorf("%s returned HTTP status: %s", scrapeUrl, resp.Status)
+		}
+
+		log.Printf("D! HTTP response content length: %d", resp.ContentLength)
```
Comment (danielnelson) [inline plugins/inputs/bind/xml_stats_v3.go]: Remove

File: plugins/inputs/bind/json_stats.go [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+			return err
+		}
+
+		defer resp.Body.Close()
+
+		if resp.StatusCode != http.StatusOK {
+			return fmt.Errorf("%s returned HTTP status: %s", scrapeUrl, resp.Status)
+		}
+
+		log.Printf("D! HTTP response content length: %d", resp.ContentLength)
```
Comment (danielnelson) [inline plugins/inputs/bind/json_stats.go]: Remove

File: plugins/inputs/bind/xml_stats_v2.go [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+		return err
+	}
+
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("%s returned HTTP status: %s", addr, resp.Status)
+	}
+
+	log.Printf("D! HTTP response content length: %d", resp.ContentLength)
```
Comment (danielnelson) [inline plugins/inputs/bind/xml_stats_v2.go]: Remove

**Comment by danielnelson** [comment]
Thanks @danielllek and @dswarbrick, it's merged!

**Pull request merged**